### ISSUE TITLE
Improvements for PBEM save game handling

### ISSUE THREAD

**Issue by hweimer** [raceintospace/raceintospace#438]
This issue tracks ideas for improvements with the handling of save games in PBEM mode:

- Show the list of already existing save games so you don't have to remember the names
- Use SaveFileHdr.Name for the actual file name
- Show PBEM files in a different color

**Comment by peyre** [comment]
Good to get these wrapped up into an Issue item.  If/once PBEM starts to show in a different color, I will be sure to update the help file so that F1 in the Time Capsule shows that file names in yellow (or whatever) indicate PBEM games.

**Comment by hweimer** [comment]
Good idea to update the help file as well. Do you have a tool to edit the help file? I once tried to add something to the help file, but gave up pretty quickly.

**Comment by peyre** [comment]
I do!  Ryan's brother built one.  It's a command-line tool but once you get the hang of it it's very straightforward and reliable.  I've already made many edits to the help file, partly to bring it into line with changes we've made.

**Comment by rnyoakum** [comment]
>I do! Ryan's brother built one. It's a command-line tool but once you get the hang of it it's very straightforward and reliable. I've already made many edits to the help file, partly to bring it into line with changes we've made.

It's available at https://github.com/gyoakum/ris-editor. The documentation needs a little work, but I'll see if he will update the docs.

**Comment by peyre** [comment]
Also, there was supposed to be a messaging system through the Tracking Station (which was never implemented).  It would be good to build that, perhaps in time for the 2.1 release (not 2.0).

**Comment by hweimer** [comment]
Thanks for the pointers to ris-editor, I wasn't aware of that.

**Comment by peyre** [comment]
I think we should add it to the project as an extra or something.

**Comment by hweimer** [comment]
I have addressed the issues mentioned in the initial comment in #449, but we should leave this issue open until the help file is adjusted as well.

**Comment by peyre** [comment]
I was just wondering.  Do you think hotseat games should also show in a different color?
